.gitignore

```diff
@@ -149,3 +149,6 @@ python/vyos/xml_ref/pkg_cache/*_cache.py
 # We do not use pip
 Pipfile
 Pipfile.lock
+
+# KDE
```

Comment:
This change might be sensible for KDE users (not for me — I use MATE, btw; but @jestabro does use KDE :).
But it's out of scope of this PR, could you make it separate please?